source/module_hamilt_lcao/module_deepks/LCAO_deepks_torch.cpp

```diff
@@ -29,6 +29,7 @@
 #include "module_hamilt_lcao/module_hcontainer/atom_pair.h"
 #include "module_base/libm/libm.h"
 #include "module_base/blas_connector.h"
+#include "module_io/input.h"
```

Comment:
this dependency is not necessary, once you have added input.h into this source file, you create a new dependence that LCAO_deepks_torch.cpp depends on the INPUT class. In fact, if you pass the variable defined in INPUT class as an input parameter, this dependence can be eliminated.

Reply:
I've explained that it does not seem absolutely necessary.